pykilosort/gui/sorter.py

```diff
@@ -70,9 +71,15 @@ def get_whitened_traces(raw_data, probe, params, whitening_matrix):
     return whitened_traces, whitening_matrix
 
 
-def get_predicted_traces(matrix_U, matrix_W, sorting_result, time_limits):
-    W = cp.asnumpy(matrix_W)
-    U = cp.asnumpy(matrix_U)
+@jit(nopython=True)
```

Comment:
if we use numba for the first time do we need to add it to our requirements (conda & pip)

Reply:
it looks like you already [added it](https://github.com/MouseLand/pykilosort/blame/a34edc8e657b6ee38147104c5726d034bb583c3d/pyks2.yml#L23) at some point last year, so we don't need to add it any more